### PR TITLE
Implement hangouts chat notifications

### LINKED ISSUE #5412: Hangouts Chat - Supported notifications
Hello everyone!
In the past couple of days I have being working on a project to make alerts be send by email and other platform (in my case Hangouts Chat). 

I studied how Netdata works with other platforms (alerta.io,Rocket.Chat,Slack,etc). Then studied how Hangouts Chat API, bots, Incoming webhooks works and make something that can be integrated in Netdata.

So I managed to create a SEND_HANGOUTS through Incoming webhooks. I take from example what was made for Rocket.Chat (the principle is the same), so I created on 'alarm-notify.sh' the code to send to Hangouts. (FYI it was my first time to create something big in shell script, so if you spot an error or something wrong fill free to comment out).

```
# -----------------------------------------------------------------------------
# hangouts sender

send_hangouts() {
 local webhook="${1}" channels="${2}" httpcode sent=0 channel color payload

 [ "${SEND_HANGOUTS}" != "YES" ] && return 1

 case "${status}" in
  WARNING) color="#ffa700" ;;
  CRITICAL) color="#d62d20" ;;
  CLEAR) color="#008744" ;;
  *) color="#777777" ;;
 esac

  for channel in ${channels}; do
  payload="$(
   cat <<EOF
    {
      "cards": [
        {
          "header": {
             "title": "Netdata on ${host}",
             "imageUrl": "${images_base_url}/images/banner-icon-144x144.png",
             "imageStyle": "IMAGE"
	},
	"sections": [
           {
             "header": "<b>${host}</b>",
             "widgets": [
                {
                  "keyValue": {
                   "topLabel": "Status Message",
                   "content": "<b>${status_message}</b>",
                   "contentMultiline": "true",
                   "iconUrl": "${image}",
                   "onClick": {
                     "openLink": {
                       "url": "${goto_url}"
                      }
                     }
                  }
                 },
               {
                "keyValue": {
                  "topLabel": "${chart} | ${family}",
                    "content": "<font color=${color}>${alarm}</font>",
                    "contentMultiline": "true"
                 }
                }
              ]
            },
            {
              "widgets": [
                {
                  "textParagraph": {
                  "text": "<font color=\"#0057e7\">@ ${date}\n<b>${info}</b></font>"
                   }
                 }
               ]
             },
             {
               "widgets": [
                 {
                   "buttons": [
                     {
                       "textButton": {
                         "text": "Go to ${host}",
                         "onClick": {
                           "openLink": {
                             "url": "http://${host}:19999/"
                           }
                         }
                        }
                       }
                     ]
                    }
                 ]
       }
     ]
    }
   ]
  }
EOF
       )"

  httpcode=$(docurl -H "Content-Type: application/json" -X POST -d "${payload}" "${webhook}")

  if [ "${httpcode}" = "200" ]; then
     info "sent hangouts notification for: ${host} ${chart}.${name} is ${status} to '${channel}'"
     sent=$((sent + 1))
  else
    error "failed to send hangouts notification for: ${host} ${chart}.${name} is ${status} to '${channel}', with HTTP error code ${httpcode}."
   fi
  done

  [ ${sent} -gt 0 ] && return 0

  return 1
}
```

This code will create a card message on Hangouts like this:
example

In the file **health_alarm_notify.conf** I only copied all settings from Rocket.Chat and changed to Hangouts.

For now, the only limitation I have is to resize the card message to fit more info horizontally, like is shown on the documentation of [configure Rocket.Chat alerts in Netdata](https://docs.netdata.cloud/health/notifications/rocketchat/).

### PR DESCRIPTION
**Pull request by hendrikhofstadt**
## Summary
fixes #5412

Add Hangouts Chat to notifications (#5412)

Originally implemented by @EnzoAkira in #5423. The previous was ready to be merged but lacked a rebase.

## Component Name
Files changed:

alarm-notify.sh
health_alarm_notify.conf

## Additional Information
Successfully tested on the current Hangouts chat Webhook API

### CODE REVIEW

File: health/notifications/hangouts/README.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Hangouts Chat
+
+This is what you will get:
+![Netdata on Hangouts](https://imgur.com/a/FVlZ3F0)
```
Comment (joelhans) [inline health/notifications/hangouts/README.md]: This image wasn't loading here on GitHub, and we prefer images be on their CDN anyway, so here's the new Markdown.

```suggestion
![Netdata on Hangouts](https://user-images.githubusercontent.com/1153921/66427166-47de6900-e9c8-11e9-8322-b4b03f084dc1.png)
```

File: health/notifications/hangouts/README.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+You need:
```
Comment (joelhans) [inline health/notifications/hangouts/README.md]: Put an empty line above this.

File: health/notifications/hangouts/README.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+1. A room or more rooms
```
Comment (joelhans) [inline health/notifications/hangouts/README.md]: "One or more rooms"

```suggestion
1. One or more rooms
```

File: health/notifications/hangouts/README.md (hunk 2 above)
Comment (joelhans) [reply health/notifications/hangouts/README.md]: And maybe clarify that you need to set these up in Google Hangouts, not Netdata? Someone might get confused. One possibility:

```suggestion
To receive notifications in Google Hangouts, you need the following in your Hangouts setup:
```

File: health/notifications/hangouts/README.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+2. In each room create a **Incoming Webhooks**
```
Comment (joelhans) [inline health/notifications/hangouts/README.md]:
```suggestion
2. An **incoming webhook** for each room
```

File: health/notifications/hangouts/README.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+
+How to create a incoming webhooks:
```
Comment (joelhans) [inline health/notifications/hangouts/README.md]:
```suggestion
How to create an incoming webhook: 
```

File: health/notifications/hangouts/README.md [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+![Netdata on Hangouts](https://imgur.com/a/FVlZ3F0)
+You need:
+
+1. A room or more rooms
+2. In each room create a **Incoming Webhooks**
+
+How to create a incoming webhooks: 
+https://developers.google.com/hangouts/chat/how-tos/webhooks
+
+Set them in `/etc/netdata/health_alarm_notify.conf` (to edit it on your system run `/etc/netdata/edit-config health_alarm_notify.conf`), like this:
```
Comment (joelhans) [inline health/notifications/hangouts/README.md]:
```suggestion
Set the webhook URIs and room names in `health_alarm_notify.conf`. To edit it on your system, run `/etc/netdata/edit-config health_alarm_notify.conf`):
```

**Comment by hendrikhofstadt** [comment]
@joelhans thanks. Applied your change requests.

**Pull request merged**